src/models/sheetmusic.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * SPDX-FileCopyrightText: 2020 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+import Sheetmusic from '../components/Sheetmusic.vue'
+
+export default {
+	id: 'sheetmusic',
+	group: 'media',
+	mimes: [
+		// These are the official mime types but nextcloud has no proper mechanism to 
+		// update existing files to match them
+		'application/vnd.recordare.musicxml',
+		'application/vnd.recordare.musicxml+xml',
+		// this registers a binary file for the sheet viewer. 
+		// The Viewer itself checks if it can make sense of it
+		'application/octet-stream', 
```

Comment:
This should not be there.
the two mime above should be enough :)

Reply:
I know, thing is that I have no idea how to persuade Nextcloud to use the application/vnd.recordare.musicxml+xml MIME type for music xml. It always shows as binary. Any suggestion?

Reply:
Absolutely!! There is a dedicated mime file on the server repository. It automatically matches an extension with a mime :)
You need to edit it here. Let me search for an example

Reply:
https://github.com/nextcloud/server/pull/42542

Here :)

Reply:
Aaaah nice. That's what I was looking for. Thought the match is done here. Ok I will submit a request there